### ISSUE TITLE
Deploy eosfilestore smart contract to EOSIO mainnet

### ISSUE THREAD

**Issue by DenisCarriere** [grigio/eosfilestore#6]
We (EOS Nation) would love to see EOSFileStore deployed under their own smart contract `eosfilestore` instead of depending on `decentwitter` to process the transactions.

@grigio Let us know via [Twitter](https://twitter.com/eos_nation)/[Telegram](https://t.me/eosnation) whenever you deploy it.

[EOS Nation Community Fund](https://eosnation.io/community-engagement-fund/) sponsored [25 EOS](https://eosflare.io/tx/247dd176f76157b36401c6a51843fdc6370036ee37eb89aa53d7b0ccf70c04f4) to help deploy your smart contract on EOSIO mainnet.

Best of luck, continue this great project!! 🚀 🚀

**Comment by grigio** [comment]
Thanks! @DenisCarriere I'll close this bug when I'll publish the smart contract in mainnet

**Comment by grigio** [comment]
Fixed in v0.1.0
`eosfilestore get 4b22bf4ff4d66e8157d22de9021431a79f696df1c0a8ab6e0c27fb3a75 Thanks.txt`

**Comment by DenisCarriere** [comment]
You're welcome! 👍 

```
$ eosfilestore get 9a9e4d3637cbecea36d7cf54d0cf8a7e8046f0b893a1d880800ec8312c7d9eb4 Welcome.txt
```
